mappings/net/minecraft/client/network/AllowedAddressResolver.mapping

```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_6370 net/minecraft/client/network/AllowedAddressResolver
+	FIELD field_33745 DEFAULT Lnet/minecraft/class_6370;
+	FIELD field_33746 addressResolver Lnet/minecraft/class_6369;
+	FIELD field_33747 redirectResolver Lnet/minecraft/class_6371;
+	FIELD field_33748 isAllowed Ljava/util/function/Predicate;
+	METHOD <init> (Lnet/minecraft/class_6369;Lnet/minecraft/class_6371;Ljava/util/function/Predicate;)V
+		ARG 1 addressResolver
+		ARG 2 redirectResolver
+		ARG 3 isBlocked
+	METHOD method_36904 getBlockListPredicate ()Ljava/util/function/Predicate;
+	METHOD method_36905 (Lcom/google/common/collect/ImmutableList;Lnet/minecraft/class_6368;)Z
+		ARG 1 address
+	METHOD method_36906 (Lnet/minecraft/class_6368;Ljava/util/function/Predicate;)Z
+		ARG 1 predicate
+	METHOD method_36907 resolveSrvAndGetAllowedAddress (Lnet/minecraft/class_639;)Ljava/util/Optional;
```

Comment:
```suggestion
	METHOD method_36907 resolve (Lnet/minecraft/class_639;)Ljava/util/Optional;
```
Since you already named this class `AllowedAddressResolver` and this is the only public api method it exposes, I think `resolve` suffices.